recipe/meta.yaml

```diff
@@ -62,9 +69,7 @@ test:
     - pytest-cov
     - boto3
     - hypothesis
-    # Workaround recent NumPy 1.24 deprecations, which cause test failures.
-    # xref: https://github.com/conda-forge/pandas-feedstock/issues/150
-    - numpy <1.24
+    - numpy
```

Comment:
Mind ensuring that some of these packages are synced with https://github.com/pandas-dev/pandas/blob/main/ci/meta.yaml?